File: lab3KononKI305/src/CallHistory.java
// CallHistory.java
import java.util.ArrayList;
import java.util.List;

public class CallHistory {
    private List<String> callHistory;

    // Default constructor
    public CallHistory() {
        callHistory = new ArrayList<>();
    }

    // Add call to the history
    public void addCall(String phoneNumber) {
        callHistory.add(phoneNumber);
        System.out.println("Call added to history: " + phoneNumber);
    }

    // Display call history
    public void displayCallHistory() {
        System.out.println("Call History:");
        for (String call : callHistory) {
            System.out.println(call);
        }
    }
}

File: lab9KononKI305/call_history.py
# call_history.py
class CallHistory:
    def __init__(self):
        self.call_history = []

    def add_call(self, phone_number):
        self.call_history.append(phone_number)
        print("Call added to history:", phone_number)

    def display_call_history(self):
        print("Call History:")
        for call in self.call_history:
            print(call)
